charts/clamav/values.yaml

```diff
@@ -106,7 +106,7 @@ nginxProxy:
   # nginxProxy.image -- The nginx proxy docker image
   image: quay.io/ukhomeofficedigital/nginx-proxy
   # nginxProxy.version -- The nginx proxy docker image version
-  version: v3.4.20
+  version: v4.2.0
```

Comment:
Should this be pinned to SHA256 as per our best practices?

Reply:
I have added the sha